components/google_cloud/sources/bigquery-new-row/bigquery-new-row.js

```diff
@@ -0,0 +1,180 @@
+const crypto = require("crypto");
```

Comment:
Will this source emit sample rows that already exist in the table on first run?

Reply:
Yes, the `deploy` hook will make sure of that.